docs/src/inputs/input/input.jsx

```diff
@@ -891,6 +897,7 @@ export default class ElementsButton extends React.Component {
                     <Input label="Email" type="email" /><br /><br />
                     <Input label="Number" type="number" min={25} max={99} style={{ width: 75 }} /> <span>Min: 25, Max: 99</span><br /><br />
                     <Input label="Number" type="number" required min={25} max={99} style={{ width: 75 }} /> <span>Min: 25, Max: 99</span><br /><br />
+                    <Input label="Number" type="number" allowDecimals required style={{ width: 75 }} /> <span>Allows Decimals</span><br /><br />
```

Comment:
Thanks for adding this example and updating the docs.
If you have spare cycles, want to "modernize" this documentation page?  (See for example the page for **Date Picker Input** to see what I mean and how it is organized).

**_Not a Blocker_** for this PR by any stretch of the imagination